application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableModerationAction.java

```diff
@@ -12,21 +12,14 @@
  */
 interface RevocableModerationAction {
     /**
-     * Classification of a revocation failure.
+     * The name of the action, that will be logged in case of a failed revocation.
```

Comment:
Either you call it `actionName` but then also give a more generic and less detailed description of its use case - or you call it `failedRecovationActionName` and only then this javadoc description is okay (I would go for former).